app/helpers/articles_helper.rb

```diff
@@ -14,6 +14,7 @@ def has_vid?(article)
 
     article.processed_html.include?("youtube.com/embed/") ||
       article.processed_html.include?("player.vimeo.com") ||
+      article.processed_html.include?("clips.twitch.tv/embed") ||
```

Comment:
can I ask you the reason for this change? I'm not super familiar with that part of the code, so I'm wondering why we need Fluidvids for a liquid tag if it's in an iframe, is it for the potential resizing of the container?